interface/client/templates/layout/browserBar.html

```diff
@@ -1,5 +1,5 @@
 <template name="layout_browserBar">
-    <div class="browser-bar {{#if TemplateVar.getFrom currentWebView 'loading'}}loading{{/if}}">
+    <div class="browser-bar {{#if TemplateVar.getFrom currentWebView 'loading'}}loading{{/if}} {{#if appBar}}url-bar-{{appBar}}{{/if}}">
```

Comment:
If you move the {{with dapp}} above this bar, then the appbar is right away available and you don't need to create that extra helper..